README.md

```diff
@@ -46,6 +46,8 @@ Okteto decouples deployment from development. Because of this, you can deploy yo
 
 Local or remote. If you can run `kubectl apply` you can use Okteto. Our community uses Okteto in all major Kubernetes distros, from Minikube and k3s all the way to GKE, Digital Ocean, AKS, EKS and Civio.
 
+You can also use Okteto in [Okteto Cloud](https://cloud.okteto.com/), the remote development platform for Kubernetes applications. Visit our [landing page](https://okteto.com/) for more details.
```

Comment:
I'd remove the 'Visit our landing page` part and change the link to okteto.com instead of cloud.okteto.com